powerhome/playbook
# [Play-118] Adding All Colors to Background Kit

![React Kit Coverage](https://img.shields.io/badge/React%20Kit%20Coverage-12.17%25-red)
#### Screens

No screen shots needed because I am only affecting the props. 

#### Breaking Changes

No breaking changes

#### Runway Ticket URL

[Play-118](https://nitro.powerhrg.com/runway/backlog_items/PLAY-118)


#### How to test this

You have to go into a playbook kit and make sure you add the specific background type you want to use. Next you can call the prop 

#### Checklist:
